docs/sql-ref-syntax-ddl-create-table-hiveformat.md

```diff
@@ -31,7 +31,13 @@ CREATE [ EXTERNAL ] TABLE [ IF NOT EXISTS ] table_identifier
     [ COMMENT table_comment ]
     [ PARTITIONED BY ( col_name2[:] col_type2 [ COMMENT col_comment2 ], ... ) 
         | ( col_name1, col_name2, ... ) ]
-    [ ROW FORMAT row_format ]
+    [ ROW FORMAT SERDE serde_name [WITH SERDEPROPERTIES (k1=v1, k2=v2, ...)] ]
```

Comment:
How about following the Databricks doc and creating a separate section for `row_format`? https://docs.databricks.com/spark/latest/spark-sql/language-manual/create-table.html

Reply:
`serde_name` -> `serde_class`